MmSupervisorPkg/Test/MmSupvRequestUnitTestApp/MmSupvRequestUnitTestApp.c

```diff
@@ -364,6 +364,7 @@ VerifyIoPolicy (
     if ((MatchFound && (AccessAttr == SMM_SUPV_ACCESS_ATTR_ALLOW)) ||
         (!MatchFound && (AccessAttr == SMM_SUPV_ACCESS_ATTR_DENY)))
     {
+      DEBUG ((DEBUG_INFO, "IoLevel error! Matchfound: %d at Index1: %d and Index2 %d\n", MatchFound, Index1, Index2));

```

Comment:
This is fine (feel free to ignore), but `MatchFound` would be a little easier to read as a BOOLEAN if printed TRUE/FALSE.

```suggestion
      DEBUG ((
        DEBUG_INFO,
        "IoLevel error! Matchfound: %a at Index1: %d and Index2 %d\n",
        MatchFound ? "TRUE" : "FALSE",
        Index1,
        Index2
        ));
```

Reply:
Updated.